tools/test_verify_versions.py

```diff
@@ -15,5 +15,9 @@ def test_unknown_tag(self):
             verify_versions.retrieve_tag_info("unknown_tag", "https://api.github.com/repos/Alystrasz/Alystrasz.Parkour/tags")
         self.assertTrue('Tag not found.' in str(context.exception))
 
+class TestModVerification(unittest.TestCase):
```

Comment:
Could you add a test evaluating `verify_all_mod_versions` result in case of failure?